.github/workflows/ci.yml

```
@@ -0,0 +1,95 @@
+          sleep 3 #
+      - name: Build and run tests with Maven
+        run: |
+          # Pass system properties to fix Chrome issues
+          mvn clean test \
+            -Dwebdriver.chrome.driver=$(which chromedriver) \
+            -Dchrome.options="--no-sandbox,--headless=new,--disable-gpu,--disable-dev-shm-usage,--remote-allow-origins=*" \
+            -Duser.data.dir=/tmp/chrome-data-$(date +%s)
+        continue-on-error: true
+
+      - name: Get Allure history
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)